deployment/media-insights-stack.yaml

```diff
@@ -129,7 +96,7 @@ Resources:
             Action:
               - sts:AssumeRole
       Policies:
-        - PolicyName: !Sub "${AWS::StackName}-stage-execution-lambda-role"
```

Comment:
Removed the "-role" since these are actually policies. Also helps reduce the chance of running into a character limit for naming.